src/install-jdk/index.ts

```diff
@@ -0,0 +1,152 @@
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { openExternalLinkFromWebview, webviewCmdLinkHandler } from '../utils';
+import { availableReleases, latestCompatibleAsset } from '../utils/adoptiumApi';
+import { WEBVIEW_ID } from './constants';
+
+
+export function installJDKCmdHandler(context: vscode.ExtensionContext, _operationId?: string) {
+	InstallJdkPage.createOrShow(context.extensionPath);
+}
+
+export class InstallJdkViewSerializer implements vscode.WebviewPanelSerializer {
+	constructor(private context: vscode.ExtensionContext) { }
+	async deserializeWebviewPanel(webviewPanel: vscode.WebviewPanel, _state: unknown) {
+		InstallJdkPage.createOrShow(this.context.extensionPath, webviewPanel);
+	}
+
+}
+
+class InstallJdkPage {
+	public static instance: InstallJdkPage | undefined;
+	private static readonly viewType = WEBVIEW_ID;
+	private readonly _panel: vscode.WebviewPanel;
+	private readonly _extensionPath: string;
+	private _disposables: vscode.Disposable[] = [];
+
+	public static createOrShow(extensionPath: string, webviewPanel?: vscode.WebviewPanel) {
+		// "smart" Beside
+		const column = vscode.window.activeTextEditor?.viewColumn !== vscode.ViewColumn.One ?
```

Comment:
Seems that viewColumn will be undefined when the active editor is a webview

Reply:
Yes, e.g. VS Code's own getting started page.     Do you have any idea to also cover these cases?

BTW, I don't like `ViewColumn.Beside` as it always creates a new Column, which is wired when I already have multiple columns.

Reply:
Have no idea how to solve this 🤔 . Maybe use `Beside` when it's undefined?

Reply:
worth a try.